pkg/apply/taskrunner/task_test.go

```diff
@@ -127,7 +127,7 @@ func TestWaitTask_CompleteEventually(t *testing.T) {
 	}()
 
 	// wait for task result
-	timer := time.NewTimer(5 * time.Second)
+	timer := time.NewTimer(500 * time.Second)
```

Comment:
Why make the timeout so big? Was it failing?
This is per test...

Reply:
Just a leftover from when I was trying to debug something. I have reverted the change.